www/include/configuration/configObject/traps/traps.php

```diff
@@ -95,23 +101,27 @@
 }
 
 switch ($o) {
-    case "a":
+    case TRAP_ADD:
         require_once($path."formTraps.php");
-        break; #Add a Trap
-    case "w":
+        break;
+    case TRAP_WATCH:
         require_once($path."formTraps.php");
-        break; #Watch a Trap
-    case "c":
+        break;
+    case TRAP_MODIFY:
         require_once($path."formTraps.php");
-        break; #Modify a Trap
-    case "m":
-        $trapObj->duplicate(isset($select) ? $select : array(), $dupNbr);
+        break;
```

Comment:
I think the 2 cases are the same requirements, so as long as there are no specific treatments for one of them I think we can group them together.
And add a space around the concatenation.

```suggestion
         case TRAP_ADD:
         case TRAP_WATCH:
         case TRAP_MODIFY:
             require_once($path . "formTraps.php");
             break;
```

Reply:
Yes, you're right